app/models/episode.rb

```diff
@@ -32,4 +32,13 @@ def knocked_out
     end
   end
 
+  def to_json
+    {
+        name: name,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.